### ISSUE TITLE
plugin.video.plutotv: Can't bypress age restriction

### ISSUE THREAD

**Issue by LouisDK1** [Lunatixz/KODI_Addons#216]
Currently it's not possible to bypass age restriction that's in action on certain content and live channels on Pluto.tv.

**Comment by Lunatixz** [comment]
Yep; the default is blocked. I'm planning on bringing back login support which will authorize adult content.
